packages/illustrations/src/assets/products/publicGateway/index.ts

```diff
@@ -0,0 +1,13 @@
+const BASE_URL = 'https://ultraviolet.s3.fr-par.scw.cloud/illustrations'
```

Comment:
would it be too hard to have BASE_URL into a contant file and we import it in every file instead of declaring it again each time? I know it's automatic, just wondering

Reply:
Maybe also, in order to prevent manual edit here, we could add a comment at top of each file explaining that each file is automatically generated when pr is opened and we shouldn't edit it here, etc..